README.md

```diff
@@ -77,6 +75,41 @@ and not the one in the production.js. Even though the NODE_ENV will be same as p
 taken from internal.js. This will help us keep the environments clean and keep thing separately for internal and
```

Comment:
This helps segregate the configuration for internal and external facets of the service.